Example/Source/Controllers/ActionsController/ActionsController.swift

```diff
@@ -10,37 +10,37 @@ import UIKit
 
 class ActionsController: ContentActionable {
     private let internalView = ActionsView()
-    
+
     // MARK: ContentActionableImplementation
-    
+
     var isShowAdvancedActions: Bool {
         get {
-            return internalView.isShowAdvancedActions
+            internalView.isShowAdvancedActions
```

Comment:
Update Swift and Xcode versions in `Requirements` section in [README.md](https://github.com/touchlane/SlideController/blob/develop/README.md)

P.S. Swift 5.1+, Xcode 11+